personal-website/src/app/navbar.js

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+
+export default function Navbar() {
+  return (
+    <nav className="navbar">
+      <h1 className="logo">
+        <Link href={"/"}>Personal Website</Link>
+      </h1>
+      <div className="navbar-links">
+        <Link href={"/"}>Wesley Tam</Link>
```

Comment:
I dont think the brackets are necessary here since only strings are being passed